runtime/compiler/control/HookedByTheJit.cpp

```diff
@@ -3968,9 +3968,9 @@ void turnOffInterpreterProfiling(J9JITConfig *jitConfig)
       }
    }
 
-/// The following two methods (stopInterpreterProfiling and restartInterpreterProfiling)
-/// are used when we disable/enable JIT compilation at runtime
-/// @{
+/** The following two methods (stopInterpreterProfiling and restartInterpreterProfiling)
+ are used when we disable/enable JIT compilation at runtime
+*/ @{
```

Comment:
The `@{` is a Doxygen grouping operator.  It should appear inside the comment, like this:

```
/**
 * The following two methods (stopInterpreterProfiling and restartInterpreterProfiling)
 * are used when we disable/enable JIT compilation at runtime
 * @{
 */
```